### DISCUSSION TITLE
Auto-run doesn't work non Steam Deck

### DISCUSSION THREAD

**Discussion by aidinzolghadr** [debauchee/barrier#1930] [General]
Exactly as the title says.

**Comment by Silveere** [comment]
I believe that setting controls whether or not the client starts as soon as you open the GUI. If you want it to run as soon as you start desktop mode, you can copy/link the desktop entry from `/var/lib/flatpak/exports/share/applications` to `~/.config/autostart` (assuming you've installed it via Flatpak/Discover):

    ln -s /var/lib/flatpak/exports/share/applications/com.github.debauchee.barrier.desktop ~/.config/autostart/
